tests/xtro-sharpie/common-AVFoundation.ignore

```diff
@@ -56,35 +56,13 @@
 !missing-release-attribute-on-return-value! CoreMedia.CMFormatDescription AVFoundation.AVTimedMetadataGroup::CopyFormatDescription()'s selector's ('copyFormatDescription') Objective-C method family ('copy') indicates that the native method returns a retained object, and as such a '[return: Release]' attribute is required.
 
 # Initial result from new rule extra-null-allowed
-!extra-null-allowed! 'AVFoundation.AVAudioConnectionPoint[] AVFoundation.AVAudioEngine::OutputConnectionPoints(AVFoundation.AVAudioNode,System.nuint)' has a extraneous [NullAllowed] on return type
-!extra-null-allowed! 'AVFoundation.AVAudioInputNode AVFoundation.AVAudioEngine::get_InputNode()' has a extraneous [NullAllowed] on return type
+!extra-null-allowed! 'Foundation.NSNumber[] AVFoundation.AVVideoCompositionInstruction::get_RequiredSourceTrackIDs()' has a extraneous [NullAllowed] on return type
```

Comment:
For some reason this is still reported as an "error" - even though both the [docs](https://developer.apple.com/documentation/avfoundation/avvideocompositioninstructionprotocol/1388661-requiredsourcetrackids) and Xcode document it as nullable on the Swift side (not on the [Obj-C side](https://developer.apple.com/documentation/avfoundation/1386654-avvideocompositioninstruction/1388661-requiredsourcetrackids?language=objc) though).

Reply:
xtro tests are based on sharpie and pre-compiled ObjC headers
we can add (keep) the ignore with a link to the documentation

Reply:
Added links to docs